### PR TITLE
get latest version of FarColorer from colorer repo

### PR DESCRIPTION
**Pull request by ctapmex**
and update to 1.4.5

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ build_script:
   - copy /Y Far-NetBox\*.md ci\%configuration%.%platform%\Plugins\NetBox
   - copy /Y Far-NetBox\LICENSE.txt ci\%configuration%.%platform%\Plugins\NetBox
   #build FarColorer
-  - set COLORER_VERSION=1.4.4
+  - for /F "delims=xxx" %a in ('curl -fsSLJ --ssl-no-revoke "https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt"') do set COLORER_VERSION=%a
```
Comment (alabuzhev) [inline appveyor.yml:111]: - `%` should be doubled in scripts, e.g. `%%a`  in both places (probably that's why it fails now).
- `"delims=xxx"` should probably be `"delims="`.

File: misc/nightly/colorer.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 function bcolorer {
   BIT=$1
   PLUGIN=FarColorer
-  COLORER_VERSION=1.4.4
+  COLORER_VERSION=$(curl -s https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt)
```
Comment (alabuzhev) [inline misc/nightly/colorer.sh:6]: curl keys & quoting should probably be the same in both cases.

File: appveyor.yml (hunk 1 above)
Comment (alexsatori) [reply appveyor.yml:111]: why do you need --ssl-no-revoke? would be better without it.

Comment (ctapmex) [reply appveyor.yml:111]: it wasn`t work without --ssl-no-revoke on local test in windows console

File: misc/nightly/colorer.sh [hunk 3]
```diff
@@ -3,12 +3,12 @@
 function bcolorer {
   BIT=$1
   PLUGIN=FarColorer
-  COLORER_VERSION=$(curl -s https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt)
+  COLORER_VERSION=$(curl -fsLJ 'https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt')
```
Comment (alexsatori) [inline misc/nightly/colorer.sh:6]: COLORER_VERSION=$(curl -fsLJ https://raw.githubusercontent.com/colorer/FarColorer/master/version4far.txt)
if [ -z "$COLORER_VERSION" ]; then
  echo "Failed to get Colorer version"
  return 1
fi

**Comment by alexsatori** [comment]
Looks good now. One last thing. For both build types - print the version so it will exist in the logs.

**Pull request merged**